.travis.yml

```diff
@@ -17,34 +19,55 @@ gemfile:
   - gemfiles/rails_5.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
 matrix:
   allow_failures:
     - rvm: ruby-head
+    - rvm: 2.4
+      gemfile: gemfiles/rails_3.gemfile
     - rvm: 2.4
       gemfile: gemfiles/rails_4.gemfile
     - rvm: 2.4
       gemfile: gemfiles/rails_4.1.gemfile
-    - rvm: 2.4
-      gemfile: gemfiles/rails_4.2.gemfile
+    - rvm: 2.5
```

Comment:
We don't need a test for Rails 4.2 on Ruby 2.4 ? Why it was removed?

Reply:
it has been removed from `allow_failures:`: this test is not supposed to fail

![image](https://user-images.githubusercontent.com/556268/57795410-28f3e780-7746-11e9-88b7-10248b5e7f1f.png)

Reply:
ref: commit message b77d9d3a9bbbeaaf005364acae209439f2eeff61

Reply:
Oh that's `allow_failures`, ✅